src/Stratis.Bitcoin.Features.RPC/Controllers/FullNodeController.cs

```diff
@@ -175,6 +177,44 @@ public BlockHeaderModel GetBlockHeader(string hash, bool isJsonFormat = true)
             return model;
         }
 
+        /// <summary>
+        /// Return information about <address>.
+        /// </summary>
+        /// <param name="address">BitcoinAddress to validate.</param>
+        /// <returns>information about <address>. {isvalid}</returns>
```

Comment:
Also not valid markup, how about:

`/// <returns>JObject containing a boolean indicating address validity.</returns>`